rpcs3/rpcs3qt/breakpoint_list.cpp

```diff
@@ -176,3 +196,82 @@ void breakpoint_list::OnBreakpointListDelete()
 		m_context_menu->close();
 	}
 }
+
+void breakpoint_list::ShowAddBreakpointWindow()
+{
+	QDialog* diag = new QDialog(this);
+
+	diag->setWindowTitle(tr("Add a breakpoint"));
+	diag->setModal(true);
+
+	QVBoxLayout* vbox_panel = new QVBoxLayout();
+
+	QHBoxLayout* hbox_top = new QHBoxLayout();
+	QLabel* l_address     = new QLabel(tr("Address"));
+	QLineEdit* t_address  = new QLineEdit();
+	t_address->setPlaceholderText("Address here");
+	t_address->setFocus();
+
+	hbox_top->addWidget(l_address);
+	hbox_top->addWidget(t_address);
+	vbox_panel->addLayout(hbox_top);
+
+	QHBoxLayout* hbox_bot = new QHBoxLayout();
+	QComboBox* co_bptype  = new QComboBox(this);
+	QStringList qstr_breakpoint_types;
+	qstr_breakpoint_types << "Memory Read"
+						  << "Memory Write"
+						  << "Memory Read&Write"
+						  << "Execution";
+	co_bptype->addItems(qstr_breakpoint_types);
+
+	hbox_bot->addWidget(co_bptype);
+	vbox_panel->addLayout(hbox_bot);
+
+	QHBoxLayout* hbox_buttons = new QHBoxLayout();
+	QPushButton* b_cancel     = new QPushButton(tr("Cancel"));
+	QPushButton* b_addbp      = new QPushButton(tr("Add"));
+
+	hbox_buttons->addWidget(b_cancel);
+	hbox_buttons->addWidget(b_addbp);
+	vbox_panel->addLayout(hbox_buttons);
+
+	diag->setLayout(vbox_panel);
+
+	connect(b_cancel, &QAbstractButton::clicked, diag, &QDialog::reject);
+	connect(b_addbp, &QAbstractButton::clicked, diag, &QDialog::accept);
```

Comment:
QDialogButtonBox* button_box = new QDialogButtonBox(QDialogButtonBox::Ok | QDialogButtonBox::Cancel);
	button_box->button(QDialogButtonBox::Ok)->setText(tr("Add"));
	connect(button_box, &QDialogButtonBox::accepted, this, &QDialog::accept);
	connect(button_box, &QDialogButtonBox::rejected, this, &QDialog::reject);